pkg/imageset/updateinstaller.go

```diff
@@ -182,6 +182,12 @@ func (o *UpdateInstallerImageOptions) Run() (returnErr error) {
 		o.log.WithField("installerImage", installerImage).Info("installer image found")
 	}
 
+	installerImagePullPolicy := string(corev1.PullIfNotPresent)
```

Comment:
This is where the default is.  I'm not sure if I should pull it up into a const to make it more visible.

Reply:
Shrug, if ya like, I'm fine either way.